### PR TITLE
adding 'fix' and a branch prefix

### PR DESCRIPTION
**Pull request by prince-chrismc**
also added a blurb for pull requests

this will allow a for a prefix to any work done

### CODE REVIEW

**Comment by DJNGattuso** [comment]
Should pull request be reviewed by all team members or should we have it to at least 1?? 

I think with all team members, it might take sometime before everyone reviews it

**Comment by prince-chrismc** [comment]
yeah I was debating about that too, thats why that section has a lot of shoulds

**Comment by DJNGattuso** [comment]
Maybe we put it to a vote? I think 1 reviewer for pull request is okay

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,16 +4,25 @@ This project is a personal exploration. This repository is the result of all the
...
 - B. New functionality which may not meet Behavoiral test criteria.
 
+The third option for a branch prefix will be `fix\`; these branches are used to directly address issues in the code and *should* be attached to an issue. This set of branches also aims to provide a space for any work or maintaince related to the project but does not meet the intentions for either `feature\` or `exploration\`. This means that no branch should exist without a prefix.
+
+### Commit Details
 While working in the correct branch, commit message must properly describe the changes applying. *Commits should compile*; however if a sitution araises where a commit does not compile it should be prefixed with a '\*' at the beginning of the message (this is excluding `master` where all commits must meet the [requirements](#Branch-Model-and-Commit-Details) above).
 
+### Pull Requests
+
+These are the key for the work and development done to transition to the final product. They should be test and reviewed throughly by *all* members of the team in a reasonable time frame. They should also have the correct labels attached in order to track which categories/domains of the project are showing progress.
```
Comment (prince-chrismc) [inline CONTRIBUTING.md]: change requirement for all members to at least 1 reviewer

Comment (prince-chrismc) [reply CONTRIBUTING.md]: this change is applied since two of three agreed

**Review by DanielPri — APPROVED** [review]
I have read and agreed to the terms and conditions

**Review by DJNGattuso — APPROVED** [review]
Seems good!

**Pull request merged**